webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/JsonSchema.scala

```diff
@@ -0,0 +1,598 @@
+/*
+*************************************************************************************
```

Comment:
Does this file belongs to this PR or the zio-json one ?

Reply:
there is different commit and yes, I used the zio-json one. It will need to be rebased, though.